Cargo.lock

```diff
@@ -454,10 +454,8 @@ checksum = "7f2c685bad3eb3d45a01354cedb7d5faa66194d1d58ba6e267a8de788f79db38"
 dependencies = [
  "android-tzdata",
  "iana-time-zone",
- "js-sys",
  "num-traits",
  "serde",
- "wasm-bindgen",
```

Comment:
I think this change was unintentional. A good reminder to check the lock file on dep updates 😄